conf/cassandra.yaml

```diff
@@ -947,10 +947,10 @@ endpoint_snitch: SimpleSnitch
 
 # controls how often to perform the more expensive part of host score
 # calculation
-dynamic_snitch_update_interval_in_ms: 100 
-# controls how often to reset all host scores, allowing a bad host to
-# possibly recover
-dynamic_snitch_reset_interval_in_ms: 600000
+dynamic_snitch_update_interval_in_ms: 100
+# controls how often this node is permitted to send a latency probe to
+# determine if previously slow nodes have recovered. Set to -1 to disable
+dynamic_snitch_latency_probe_interval_in_ms: 1000
```

Comment:
Add dynamic snitch class name as a commented out option?

Reply:
I'm resolving this since I think we agreed not to do this.